app/views/layouts/application.html.erb

```diff
@@ -25,8 +25,6 @@
 
 	<% if current_user.present? && current_user.admin? %>
 		<%= render partial: "./partials/admin_footer" %>
-  <% elsif current_user.present? && current_user.supplier? %>
-    <%= render partial: "./partials/supplier_footer" %>
```

Comment:
Good deletion. I think we will probably want to get rid of the admin footer too.